documentation/docs/dictionary/Dictionary/Dictionary.mdx

```diff
@@ -8,12 +8,6 @@ import {Fiddle} from '@site/src/components/fiddle';
 
 # `Dictionary()`
 
-:::info
-
-This Class is deprecated, it has been renamed to [`ConfigStore`](../../fastly:config-store/ConfigStore/ConfigStore.mdx) and can be imported via `import { ConfigStore } from 'fastly:config-store'`
```

Comment:
We need to keep the first part of this, as it's still deprecated, it's just no longer related to ConfigStore.

Reply:
reverted in b6050674296c8ea9cc93cb19a20de30ebefc2ce9